crates/bindings/src/rt.rs

```diff
@@ -217,7 +217,7 @@ macro_rules! impl_reducer {
             Ret: IntoReducerResult
         {
             #[allow(non_snake_case)]
-            fn invoke(&self, _ctx: ReducerContext, args: ($($T,)*)) -> Result<(), Box<str>> {
+            fn invoke(&self, _ctx: &ReducerContext, args: ($($T,)*)) -> Result<(), Box<str>> {
```

Comment:
I believe this branch is no longer valid because the `ctx` arg is now mandatory.

Reply:
Ah, I missed that